easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-5.2.0-binutils-2.25.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.25'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+toolchain = {'name': 'GCC', 'version': '5.2.0-binutils-%(version)s'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked instatically, so not a runtime dep
+    ('zlib', '1.2.8'),
+    # use same binutils version that was used when building GCC toolchain, to 'bootstrap' this binutils
+    ('binutils', version, '', True)
+]
+
+# statically link with zlib, to avoid runtime dependency on zlib
+preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
+prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
+
+# make sure that system libraries are also considered by ld and ld.gold is also built
+# --enable-plugins should be used whenever --enable-gold is used, see http://llvm.org/docs/GoldPlugin.html
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default --enable-plugins'
```

Comment:
@wpoely86 `--enable-lto` is set by default. At least, that's what the [GCC configure docu says](https://gcc.gnu.org/install/configure.html).